### PR TITLE
Remove refs in docs and unit tests

### PR DESCRIPTION
**Pull request by andrewhughes101**
Based on https://github.com/ansible-collections/ibm_zos_cics/pull/141 so merge after that

Removes refs to addresses in units and docs and replaces with a generic example.com

Doc extractor changed the rsts a little more than I would expect but after building and viewing locally it seemed to be fine

### CODE REVIEW

File: docs/source/modules/td_intrapartition.rst [hunk 1]
```diff
@@ -20,9 +20,9 @@ td_intrapartition -- Create and remove the CICS transient data intrapartition da
 
 Synopsis
 --------
-- Create and remove the \ `transient data intrapartition <https://www.ibm.com/docs/en/cics-ts/latest?topic=data-defining-intrapartition-set>`__\  data set used by a CICS® region. This data set holds all the data for intrapartition queues.
```
[the remark this answers is not in the document]
Comment (andrewhughes101) [reply docs/source/modules/td_intrapartition.rst:23]: Not a manual thing, this is just how doc extractor does the extraction of the doc strings from the modules

File: docs/source/modules/cmci_create.rst [hunk 2]
```diff
@@ -223,9 +222,9 @@ Examples
    
    - name: define a BUNDLE in a CSD
      cmci_create:
-       cmci_host: "winmvs2c.hursley.ibm.com"
-       cmci_port: 10080
-       context: "iyk3z0r9"
+       cmci_host: "example.com"
+       cmci_port: 12345
+       context: "applid"
```
[the remark this answers is not in the document]
Comment (andrewhughes101) [reply docs/source/modules/cmci_create.rst:227]: Yeah I'm wondering if maybe `cpsm_context` is better?

[the remark this answers is not in the document]
Comment (Tom-Latham) [reply docs/source/modules/cmci_create.rst:227]: It's the name of the region so I was trying to suggest 8 character names. I'm not sure cpsm_context gives much more of a clue what to put there

[the remark this answers is not in the document]
Comment (andrewhughes101) [reply docs/source/modules/cmci_create.rst:227]: Is context always region name, was thinking in the case of a plex is it not the plex name and then the region name can be a scope?

[the remark this answers is not in the document]
Comment (Tom-Latham) [reply docs/source/modules/cmci_create.rst:227]: Good point! Maybe just `plexname` then?

**Review by Tom-Latham — APPROVED** [review]

**Pull request merged**